### PR TITLE
Add maven dependency

### PR DESCRIPTION
**Pull request by jasonkatonica**
This update adds maven as a dependency to build and test the OpenJCEPlus project. As part of this effort other dependencies that were used in the past for this project have been removed.

Signed-off-by: Jason Katonica <katonica@us.ibm.com>

### CODE REVIEW

**Comment by jasonkatonica** [comment]
This update is dependent upon https://github.com/adoptium/aqa-tests/pull/5479

File: scripts/getDependencies.pl [hunk 1]
```diff
@@ -184,55 +184,10 @@
...
-	junit_jupiter_params => {
-		url => 'https://repo1.maven.org/maven2/org/junit/jupiter/junit-jupiter-params/5.10.2/junit-jupiter-params-5.10.2.jar',
-		fname => 'junit-jupiter-params.jar',
-		sha1 => '359132c82a9d3fa87a325db6edd33b5fdc67a3d8'
-	},
-	junit_platform_suite_api => {
-		url => 'https://repo1.maven.org/maven2/org/junit/platform/junit-platform-suite-api/1.10.2/junit-platform-suite-api-1.10.2.jar',
-		fname => 'junit-platform-suite-api.jar',
-		sha1 => '174bba1574c37352b0eb2c06e02b6403738ad57c'
+	maven => {
```
Comment (karianna) [inline scripts/getDependencies.pl:187]: can we go to 3.9.8?

**Review by karianna — CHANGES_REQUESTED** [review]

File: scripts/getDependencies.pl (hunk 1 above)
Comment (jasonkatonica) [reply scripts/getDependencies.pl:187]: Yes version `3.9.8` will work for us. Updated to this version using https://github.com/adoptium/TKG/pull/591/commits/8a8b966894abd79eedc5b304e110012d5e52e3aa

**Review by karianna — APPROVED** [review]

**Comment by llxia** [comment]
Checksum is skipped.

```
downloading https://dlcdn.apache.org/maven/maven-3/3.9.8/binaries/apache-maven-3.9.8-bin.tar.gz
--> file downloaded to ./apache-maven-bin.tar.gz
Checksum verification skipped for ./apache-maven-bin.tar.gz
```

[Internal job](https://hyc-runtimes-jenkins.swg-devops.com/job/test.getDependency/11103/console)

`sha512` is wrong as there is no logic checking for `sha512`.
https://github.com/adoptium/TKG/blob/8a8b966894abd79eedc5b304e110012d5e52e3aa/scripts/getDependencies.pl#L190

I think we should do something similar to https://github.com/adoptium/TKG/blob/8a8b966894abd79eedc5b304e110012d5e52e3aa/scripts/getDependencies.pl#L130-L132
change `shaalg => '256'`  to `shaalg => '512'`

**Review by gdams — APPROVED** [review]

**Comment by jasonkatonica** [comment]
Hi @llxia I attempted via commits https://github.com/adoptium/TKG/pull/591/commits/d07ba4683be0a6506aba6177bef289638b8739ea and https://github.com/adoptium/TKG/pull/591/commits/6867f76a6890eea07cbceb211291967daf58d902 and https://github.com/adoptium/TKG/pull/591/commits/a00e9f801164c87fca0ba45eb50a5c96983c688f to directly use the SHA512 hash file published on the website ran into issues with it recognizing either the hash value or the value of the file itself. My guess is that the URL does not end in .txt and is using 512 making it different ? In the end I simply placed the sha1 hash of the file into script for validation which seems to work now when I run it:
```
07:52:47       [exec] --------------------------------------------
07:52:47       [exec] Starting download third party dependent jars
07:52:47       [exec] --------------------------------------------
07:52:47       [exec] downloading dependent third party jars to C:/Users/jenkins/testDependency/lib
07:52:47       [exec] downloading https://dlcdn.apache.org/maven/maven-3/3.9.8/binaries/apache-maven-3.9.8-bin.tar.gz
07:52:47       [exec] --> file downloaded to C:/Users/jenkins/testDependency/lib/apache-maven-bin.tar.gz
07:52:47       [exec] downloaded dependent third party jars successfully
```

**Comment by llxia** [comment]
Thanks for making it work.

**Review by llxia — APPROVED** [review]
LGTM

**Comment by llxia** [comment]
https://openj9-jenkins.osuosl.org/job/test.getDependency/1211/

**Pull request merged**